lib/winapi/minwinbase.h

```diff
@@ -2,6 +2,7 @@
 #define __MINWINBASE_H__
 
 #include <windef.h>
+#include <xboxkrnl/ntstatus.h>
```

Comment:
Shouldn't this part of windef or xboxdef?

*(I didn't check this, but just a gut feeling)*

Reply:
There doesn't seem to be a "correct" way to include it. minwinbase has the `EXCEPTION_*` constants, so the values are required here, but while MS's headers have a `ntstatus.h`, it doesn't get included, instead `winnt.h` has the same constants.